Assets/Scripts/GrindingGame/DecoratorCode/TimeDecorator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TimeDecorator : MonoBehaviour
```

Comment:
Is this attached to a gameobject in the main scene and does it serve a purpose? Just curious since there's nothing in it?